packages/ui/src/components/ui/tabs.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react"
+import * as TabsPrimitive from "@radix-ui/react-tabs"
+
+import { cn } from "@/lib/utils"
+
+function Tabs({
+  className,
+  ...props
+}: React.ComponentProps<typeof TabsPrimitive.Root>) {
+  return (
+    <TabsPrimitive.Root
+      data-slot="tabs"
+      className={cn("flex flex-col gap-2", className)}
+      {...props}
+    />
+  )
+}
+
+function TabsList({
+  className,
+  ...props
+}: React.ComponentProps<typeof TabsPrimitive.List>) {
+  return (
+    <TabsPrimitive.List
+      data-slot="tabs-list"
+      className={cn(
+        "inline-flex rounded-none bg-transparent p-0",
+        className
+      )}
+      {...props}
+    />
+  )
+}
+
+function TabsTrigger({
```

Comment:
I think this one is slightly different than the design, the active tab is rounded and touches the bottom in design, in the code it's not touching the bottom

| Design    | Code |
| -------- | ------- |
| ![image](https://github.com/user-attachments/assets/3a9eef55-d1a1-4c53-9553-e339a9ff3c57) | ![image](https://github.com/user-attachments/assets/b7ed6118-a5c0-4800-8252-dcc95bf9b69c) |